.github/workflows/publish.yaml

```diff
@@ -7,8 +7,8 @@ on:
       - "*"
 
 jobs:
-  publish:
-    name: GoReleaser
+  publish-cli:
+    name: Publish Suga CLI
```

Comment:
Is it pushing the CLI or the node client?

Reply:
That's should just be renaming the existing job, since we publish multiple assets from this repo

Reply:
This specific job is publishing the CLI, the next one is publishing the node-sdk